src/frontend/components/Chat/SharedChatComponents/ChatMessageItem/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box } from 'grommet';
+import React from 'react';
+import { ChatMessageType } from '../../../../data/stores/useMessagesStore/index';
+import { ChatMessageContent } from '../ChatMessageContent';
+import { ChatMessageMetadatas } from '../ChatMessageMetadatas';
+
```

Comment:
please change the paths
```suggestion
import { ChatMessageType } from '../../../../data/stores/useMessagesStore/index';
import { ChatMessageContent } from '../ChatMessageContent';
import { ChatMessageMetadatas } from '../ChatMessageMetadatas';

```

Reply:
fix in [30417701d10c51da927b5c824ee5a16a5259c4a5](https://github.com/openfun/marsha/pull/1247/commits/30417701d10c51da927b5c824ee5a16a5259c4a5)